rapid/ci/data/commit_dal.py

```diff
@@ -88,7 +88,7 @@ def _create_git_commit(self, commit_identifier, additional_info, pipeline_instan
         if commit_identifier:
             commit = self._get_or_create_commit(commit_identifier, session, vcs_id)
             if additional_info:
-                for key, value in additional_info.iteritems():
```

Comment:
I didn't realize this stopped working in python3 interesting

Reply:
They renamed it. The method iteritems is now items.